docs/runtime/shell.md

```diff
@@ -108,6 +108,34 @@ The following JavaScript objects are supported for redirection from:
 - `Bun.file(path)`, `Bun.file(fd)` (reads from the file)
 - `Response` (reads from the body)
 
+### To/From Files
```

Comment:
Maybe a section on `Redirects`?

And then

### Redirecting stdout → file

### Redirecting stderr → file

### Redirecting stdin ← file

### Redirecting stdout → stderr

### Redirecting stdout ← stderr

Reply:
basically, people might not immediately make the mental leap that  stdout and stderr are a "file" 

Also, we could do buffers in there too.